projects/ngx-formentry/src/components/file-upload/file-upload.component.ts

```diff
@@ -34,14 +39,29 @@ export class RemoteFileUploadComponent implements OnInit, ControlValueAccessor {
     constructor(private renderer: Renderer2) { }
 
     ngOnInit() {
+        if (this.innerValue) {
+            const re = /pdf/gi;
```

Comment:
Refactor this into a function, and name it in a way that explains what is happening here